gui/src/handle_args/HandleArgs.cpp

```diff
@@ -0,0 +1,81 @@
+/*
+** EPITECH PROJECT, 2024
+** Zappy [WSL: kali-linux]
+** File description:
+** HandleArgs
+*/
+
+#include "HandleArgs.hpp"
+#include <string>
+#include <sstream>
+#include <vector>
+#include <cctype>
+#include <cstdlib>
+#include <iostream>
+
+using namespace GUI;
+
+int HandleArgs::checkArgs(int nbArgs, char **args)
+{
+    try {
+        if (nbArgs == 2 &&
+            (std::string(args[1]) == "-help" || std::string(args[1]) == "-h")) {
+            printUsage();
+            goto end;
```

Comment:
![falling](https://github.com/epitech-mirroring/Zappy/assets/38489683/96752435-a6fe-4466-80c0-f6692c5009cb)

Pleases remove goto, you can return here directly if you want to do so